src/Momento.Sdk/Auth/ICredentialProvider.cs

```diff
@@ -2,9 +2,21 @@ namespace Momento.Sdk.Auth;
 
 using System.Collections.Generic;
 
+/// <summary>
+/// Interface for credentials
+/// </summary>
 public interface ICredentialProvider
 {
+    /// <summary>
+    /// JWT provided by user
+    /// </summary>
     string AuthToken { get; }
-    string ControlEndpoint {get; }
+    /// <summary>
+    /// Control endpoint extracted from JWT
+    /// </summary>
+    string ControlEndpoint { get; }
+    /// <summary>
+    /// Cache endpoint extracted from JWT
+    /// </summary>
```

Comment:
In the future, the endpoints won't be embedded in the JWT, so these interface comments should probably be more general. I'd suggest we just use something like "Momento authentication token", "Control endpoint", and "Cache endpoint" as the summaries for these.

Reply:
👍 
As I was adding comments, I wasn't sure how descriptive we want those properties to be, especially these are hard to describe more than "auth token" and "endpoints" 😄